codecell/gears-shop
# #Prepare project for Review

**What does this PR do?**
it prepares the codebase for it to be reviewed  for approval

**How can this be manually tested ?**
once this project is cloned, or once you navigate to the index.html and open it in a browser, at the minimum the following pages should be loaded fully;
-The homepage auto-loaded on visiting the index.html file
-the Latest deals page can be accessed from the **wine-background button**  and/or from the  **Latest deals navbar link**  on the homepage.
- The **product page**  can be accessed by  **clicking on any image on the Latest-deals page** and by/or by clicking on any image on the **Top deals section on the  Homepage**

*Any Background context***
-Navbar behaviour on mobile screen sorted
